src/components/AddressInfo.css

```
@@ -37,6 +38,28 @@
     transform: scale(1.1);
 }
 
+.address-info .identicon .multisig-badge {
+    position: absolute;
+    right: -0.875rem;
+    bottom: -0.25rem;
+    border-radius: 5rem;
+    font-weight: bold;
+    font-size: 1.75rem;
+    padding: 0.375rem 0.75rem;
+    line-height: 1;
+    border: 0.5rem solid white;
+    word-spacing: -0.071428571em; // 1/14
```

Invalid CSS comment (should be `/* 1/14 */`). But probably better as `calc(-1em / 14)` anyways.
```suggestion
    word-spacing: calc(-1em / 14);
```

---

Resolved in #524